ote/src/cljs/ote/views/place_search.cljs

```diff
@@ -104,6 +107,7 @@
         [leaflet/Map {;;:prefer-canvas true
                       :ref "leaflet"
                       :center #js [65 25]
+                      :zoomControl false
```

Comment:
I assume, that this doesn't  prevent zooming but hides default zoom buttons?

Reply:
Yes, it allows us to use customized zoom controller.